package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "github-langs",
-  "version": "2.0.16",
```

Comment:
Please don't downgrade the version

Reply:
Sorry, I didn't know that the code was updated because I fetched the code this morning.

Reply:
ok please refetch again

Reply:
👍